app/routes.py

```diff
@@ -18,11 +18,12 @@ def index():
 #Create challenge page
 @app.route('/create', methods=['GET', 'PUT'])
 def create():
-    #TODO
+    form = LoginForm()
+    create_form = ChallengeCreationForm()
     if request.method == "PUT":
```

Comment:
this line is actually unnecessary for now believe, flask will check the next if statement anyway